x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -103,7 +92,7 @@ export const create = async (
         username,
         full_name,
         email,
-        connector: transformCaseConnectorToEsConnector(query.connector ?? caseConfigureConnector),
```

Comment:
The `transformCaseConnectorToEsConnector` has been moved into the service layer (`postNewCase`) and will occur directly before the saved object client is called to create the case.